dvc.yaml

```diff
@@ -47,7 +47,7 @@ stages:
         cache: false
     - output/workflow/recipe/model_workflow_recipe.rds:
         cache: false
-    frozen: true
```

Comment:
I'm pretty sure this step was erroneously marked as `frozen`; we don't cache intermediate outputs so freezing the `train` step leads to an error. Billy seemed to think this was reasonable but let me know if I'm off base!

Reply:
This is definitely an error. The only step that should really ever be frozen is the ingest step.